webpack/assets/javascripts/charts/c3DonutChartService.js

```diff
@@ -0,0 +1,93 @@
+import * as c3 from 'c3';
+
+const enums = {
+    SIZE: {
+        SMALL: {height: 240},
+        LARGE: {height: 500}
+    },
+    WIDTH: {
+        SMALL: 15,
+        LARGE: 25
+    }
+};
+
+export default {
+    getDonutConfig:  getDonutConfig,
+    getLargeDonutConfig: getLargeDonutConfig,
+    generate: generate
+}
+
+function getDonutConfig (columns, selector, dataEventHandlers) {
+
+    let config = {
+        bindto: selector,
+        donut: {
+            width: enums.WIDTH.SMALL,
+            label: {show: false}
+        },
+        data: {
+            type: 'donut',
+            columns: columns
+        },
+        tooltip: {
+            show: true,
+            contents: $().pfDonutTooltipContents
+        },
+        size: enums.SIZE.SMALL,
+        legend: {show: false},
+        padding: {
+            top: 0,
+            left: 0,
+            right: 0,
+            bottom: 0
+        }
+    };
+
+    if (dataEventHandlers)
+        _.assign(config.data, dataEventHandlers);
+
+    return config;
+}
+
+function getLargeDonutConfig (columns, selector, dataEventHandlers) {
+    let config = getDonutConfig(columns, selector, dataEventHandlers);
+
+    config.size = enums.SIZE.LARGE;
+    config.donut.width = enums.WIDTH.LARGE;
+
+    return config;
+}
+
+function generate (config) {
```

Comment:
nitpick: you are inconsistent with having or not having a whitespace between the function name and the parameters.